README.md

```diff
@@ -13,6 +13,14 @@ The tests depend on having Firefox (default) or Chrome downloaded.
 1. `bundle install`
 1. `bin/rake webdrivers:geckodriver:update`
 
+### Install the Playwright CLI
+
+```
+export PLAYWRIGHT_CLI_VERSION=$(bundle exec ruby -e 'require "playwright"; puts Playwright::COMPATIBLE_PLAYWRIGHT_VERSION.strip')
+yarn add -D "playwright@$PLAYWRIGHT_CLI_VERSION"
+yarn run playwright install
```

Comment:
Is this a one-time thing or should we be running it after we `bundle install` (in case the `playwright` gem bumps)?